### PR TITLE
Tl 09 homework

### PR DESCRIPTION
**Pull request by Marusia2**

### CODE REVIEW

**Review by alekseigolov1 — APPROVED** [review]
Good job ! Main task that was asked is completed , approving your PR . 
However try you best to add pre-commit hook and try to add Axios test as well .

File: tests/simple-solution-api.spec.ts [hunk 1]
```diff
@@ -31,3 +40,179 @@ test('post order with correct data should receive code 201', async ({ request })
...
+//       headers: requestHeaders,
+//     });
+//   }
+//
+//   const response = await request.delete('https://backend.tallinn-learning.ee/test-orders/1', {
+//     headers: requestHeaders,
+//   });
+//   console.log('response status:', response.status())
+//   expect(response.status()).toBe(StatusCodes.TOO_MANY_REQUESTS)
+// });
```
Comment (vladimir-ovodenko-inbank) [inline tests/simple-solution-api.spec.ts]: pls delete unsued lines

File: tests/simple-solution-api.spec.ts [hunk 2]
```diff
@@ -31,3 +40,179 @@ test('post order with correct data should receive code 201', async ({ request })
...
+// GET request. Login with missing username and password data should receive code 500
+test('Get order with missing data should receive code 500', async ({ request }) => {
+  const response = await request.get('https://backend.tallinn-learning.ee/test-orders', {})
+
+  console.log('response status:', response.status())
+  console.log('response body:', await response.json())
+  expect(response.status()).toBe(StatusCodes.INTERNAL_SERVER_ERROR)
+})
+
+// // Negative scenario with missing data  -> NOT working
```
Comment (vladimir-ovodenko-inbank) [inline tests/simple-solution-api.spec.ts]: delete

**Comment by Marusia2** [comment]
> Good job ! Main task that was asked is completed , approving your PR . However try you best to add pre-commit hook and try to add Axios test as well .

Thank you for you comment, next time will do my best :)

**Pull request merged**